### ISSUE TITLE
YAML example global config yields errors

### ISSUE THREAD

**Issue by serhez** [pvolok/mprocs#31]
Hello!

I'm trying to globally customise `mprocs` through `~/.config/mprocs/mprocs.yaml`. I started by using the example config you have in the README:

```
keymap_procs: # keymap when process list is focused
  <C-q>: { c: toggle-focus }
  <C-a>: null # unbind key
keymap_term: # keymap when terminal is focused
  reset: true
  <C-q>: { c: toggle-focus }
```

However, this yields the following error when trying to run `mprocs`:

```
Error: [global settings] invalid type: unit value, expected internally tagged enum AppEvent
```

I haven't worked with `rust` before, but I think it's coming from the `serde_yaml` library, triggered at `src/main.rs:68` <- `src/settings.rs:40` <- `src/settings.rs:86`.

**Comment by pvolok** [comment]
The problem is in this line: `<C-a>: null`. I recently added unbinding keys, but didn't release the new version yet. I'll make a new release today or tomorrow. For now you can remove that like and it should work.

**Comment by serhez** [comment]
Perfect, thanks!

**Comment by pvolok** [comment]
Now this feature is released in mprocs 0.4.1.

**Comment by serhez** [comment]
This was indeed resolved, thanks!
